app/ui/lib/CreateButton.tsx

```diff
@@ -6,15 +6,18 @@
  * Copyright Oxide Computer Company
  */
 
+import cn from 'classnames'
 import { Link, type LinkProps } from 'react-router-dom'
 
 import { AddRoundel12Icon } from '@oxide/design-system/icons/react'
 
 import { Button, buttonStyle, type ButtonProps } from '~/ui/lib/Button'
 
-export const CreateButton = ({ children, ...props }: ButtonProps) => (
-  <Button size="sm" className="shrink-0" {...props}>
-    <AddRoundel12Icon className="mr-2 text-accent-secondary" />
+export const CreateButton = ({ children, disabled, ...props }: ButtonProps) => (
+  <Button size="sm" className="shrink-0" disabled={disabled} {...props}>
```

Comment:
Because we're now pulling the `disabled` prop out of the `...props` spread on line 16, we need to add it in as an explicit prop in the `Button` component on 17.

Reply:
Another way would be to not pull it out and use `props.disabled` on line 19. either way is fine